static/js/popovers.js

```diff
@@ -136,6 +136,20 @@ exports.hide_actions_popover = function () {
     }
 };
 
+exports.topic_ops = {
+    mute: function (stream, topic) {
+        popovers.hide_topic_sidebar_popover();
+        muting.mute_topic(stream, topic, true);
+        muting_ui.persist_and_rerender();
+        muting_ui.mute_notification(stream, topic);
+    },
+    unmute: function (stream, topic) {
+        popovers.hide_topic_sidebar_popover();
+        muting.unmute_topic(stream, topic, true);
+        muting_ui.persist_and_rerender();
```

Comment:
optional suggestion: Add comment here explaining why we don't notify on the unmute path.  If you can't think of a concise reason, probably don't worry about it; people will figure it out.